app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ProfilesController < ApplicationController
+   def show
+   end
+
+   def update
+      redirect_to dashboard_path if current_user.update profile_params
```

Comment:
@jasonoverby I'd probably make this a more explicit `if`/`else` sort of thing, with a redirect for if the profile-update fails as well. As it stands, this will probably try to render an `update` template?

Also: is this needed, given that profiles get updated through the `User`? (I think this was the confusing thing you mentioned on Sunday?)